test/e2e/storage/testsuites/provisioning.go

```diff
@@ -88,10 +88,9 @@ func createProvisioningTestInput(driver TestDriver, pattern testpatterns.TestPat
 			ClaimSize:    resource.claimSize,
 			ExpectedSize: resource.claimSize,
 		},
-		cs:    driver.GetDriverInfo().Config.Framework.ClientSet,
-		pvc:   resource.pvc,
-		sc:    resource.sc,
-		dInfo: driver.GetDriverInfo(),
```

Comment:
Can we remove the field entirely?

Reply:
Which field? I'm sorry, but it's not obvious in the Github UI.

Reply:
Np. `dInfo`

Reply:
But dInfo is gone already, I'm removing it in this PR already because it was clearly redundant.

Is there anything else you want to get removed?

Reply:
https://github.com/kubernetes/kubernetes/pull/72434 removes the entire struct holding these fields.